.github/workflows/ci.yml

```diff
@@ -173,7 +172,9 @@ jobs:
       - name: "Checking out repository"
         uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
         with:
-          submodules: true
+          submodules: false
```

Comment:
Can you add a comment on why we do this manually here?

Reply:
Sure, added a comment.